gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -412,19 +423,71 @@ hook.Add("CalcView", "TTT2ViewBobbingHook", function(ply, origin, angles, fov)
         return
     end
 
-    if (not ply:IsOnGround() and ply:WaterLevel() == 0) or ply:InVehicle() then
-        airtime = math.Clamp(airtime + 1, 0, 300)
-
-        return
-    end
-
     local view = {
         ply = ply,
         origin = origin,
         angles = angles,
         fov = fov,
     }
 
+    ply.lastFOVValue = ply.lastFOVValue or fov
```

Comment:
Isn't this always the current FOV? Because we did not yet apply the next fov change?

Reply:
Only the first time the hook is called fov is used as the value, after that the lastFOVValue is used.

Reply:
No i am talking about the general value of this is baically always the previous fov value right? Why bother creating a variable when this is already available via the current fov on the player

Reply:
you mean I should use `ply:GetFOV()` here? This might work, I'm not sure if this is influenced by GMods own transitioning. Also I'm not setting the previous FOV value in all cases, there are some cases where it is intentionally not changed. So I think the getter function probably won't work here